daemon/mgr/container_exec.go

```diff
@@ -86,19 +102,11 @@ func (mgr *ContainerManager) StartExec(ctx context.Context, execid string, confi
 	c.Unlock()
 
 	execConfig.Running = true
-	defer func() {
-		if err != nil {
-			execConfig.Running = false
-			exitCode := 126
-			execConfig.ExitCode = int64(exitCode)
-		}
-		mgr.ExecProcesses.Put(execid, execConfig)
```

Comment:
I think `mgr.ExecProcesses.Put(execid, execConfig)` is unnecessary， since it has added in `CreateExec`, WDYT, @fuweid

Reply:
@allencloud , @HusterWan , does `mgr.ExecProcesses.Put(execid, execConfig)` can be removed here?

Reply:
@Ace-Tang I do not think so, we will use `exec/EXEC_ID/json` to judge the exec result, so we should update the `execConfig` when exec done